.github/workflows/dart.yml

```
@@ -65,5 +60,5 @@ jobs:
         run: |
           export PATH="$PATH:/tmp/bin"
           melos exec --file-exists="coverage/lcov.info" --concurrency=1 -- "
-            codecov --verbose upload-process --fail-on-error -F "$MELOS_PACKAGE_NAME" -f "$MELOS_PACKAGE_PATH/coverage/lcov.info" -t "${{ secrets.CODECOV_TOKEN }}"
+            codecov --verbose upload-process --fail-on-error -F "MELOS_PACKAGE_NAME" -f "MELOS_PACKAGE_PATH/coverage/lcov.info" -t "${{ secrets.CODECOV_TOKEN }}"
```

```suggestion
            codecov --verbose upload-process --fail-on-error -F MELOS_PACKAGE_NAME -f MELOS_PACKAGE_PATH/coverage/lcov.info -t ${{ secrets.CODECOV_TOKEN }}
```
The whole command is already escaped, so these additional quotes actually unquote the variables. They can be completely removed to avoid this.